components/scream/src/diagnostics/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+set(DIAGNOSTIC_SRCS
+  potential_temperature.cpp
+)
+
+set(DIAGNOSTIC_HEADERS
+  potential_temperature.hpp
+) 
+
+add_library(diagnostics ${DIAGNOSTIC_SRCS})
+target_include_directories(diagnostics PUBLIC ${CMAKE_CURRENT_SOURCE_DIR}/../share)
+target_link_libraries(diagnostics scream_share)
+target_compile_options(diagnostics PUBLIC)
```

Comment:
What's this line for? It seems to me it can be removed, since it adds no compile options...

Reply:
I stole this line from somewhere else.  So not sure.